packages/core/src/domain/oid.ts

```diff
@@ -64,6 +64,7 @@ export const driversLicenseURIs: { [k in USState]: string } = {
   OK: `${OID_PREFIX}2.16.840.1.113883.4.3.40`,
   OR: `${OID_PREFIX}2.16.840.1.113883.4.3.41`,
   PA: `${OID_PREFIX}2.16.840.1.113883.4.3.42`,
+  PR: `${OID_PREFIX}2.16.840.1.113883.4.3.100`,
```

Comment:
The Commonwell data has puerto rico in it as a state and I was getting TS errors. This is a placeholder - what is the context for these OIDs? I can't seem to grok it from poking around in the code.

Reply:
These are to create strong/personal identifiers based on the driver's license. I suggest we don't touch this and instead treat the State on the HIE entries as string, since those are read only and coming from external systems.

Reply:
addressed